query-service/src/integrationTest/resources/attribute-expression-test-queries/query2.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
Can we also add one more query say `query3.json` with selection having map attribute expression w/o subpath?

Reply:
We should also add test for `NOT_CONTAINS_KEY`.

Reply:
Please add these tests in followup

Reply:
added

Reply:
added